src/main/java/cam72cam/immersiverailroading/Config.java

```diff
@@ -101,7 +104,8 @@
 			Items.EMERALD.getRegistryName().toString()
 		};
 		
-		@Comment("Fuels for diesel Locomotives")
+		@Comment("Fuels for diesel Locomotives" + 
```

Comment:
What does the \n look like in the in-game UI and in the config text?

Reply:
![image](https://user-images.githubusercontent.com/22636066/44359345-e30c3a80-a4b7-11e8-85c6-ecdfdc651fc3.png)

Reply:
and it doesn't show me comments in the in-game UI